generate-rest-docs build step failing

currently the build fails when not running it with `-Dmaven.javadoc.skip=true` (as is also done in the travis CI build).

error output (shortened):
```
[INFO] --- bitbucket-maven-plugin:8.0.0:generate-rest-docs (default-generate-rest-docs) @ parameterized-builds ---
[WARNING] The POM for com.atlassian.plugins.rest:atlassian-rest-doclet:jar:2.9.2 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 23.265 s
[INFO] Finished at: 2019-06-24T14:08:49+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.atlassian.maven.plugins:bitbucket-maven-plugin:8.0.0:generate-rest-docs (default-generate-rest-docs) on project parameterized-builds: An error has occurred in Javadoc report generation: Unable to resolve artifact:groupId = 'com.atlassian.plugins.rest'
[ERROR] artifactId = 'atlassian-rest-doclet'
[ERROR] version = '2.9.2': Failure to find com.atlassian.plugins.rest:atlassian-rest-doclet:jar:2.9.2 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,4 +4,3 @@ jdk:
 cache:
   directories:
   - $HOME/.m2
-script: mvn test -B -Dmaven.javadoc.skip=true

--- a/src/main/resources/atlassian-plugin.xml
+++ b/src/main/resources/atlassian-plugin.xml
@@ -96,7 +96,6 @@
       <param-name>extension.filter.excludes</param-name>
       <param-value>.*</param-value>
     </init-param>
-    <package>com.kylenicholls.stash.parameterizedbuilds.rest</package>
   </rest>
   
   <!-- web items -->

